### PR TITLE
 Navigate to API console after token generation from Wizard

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
When subscribing to API from a new app using the wizard, provide an option to navigate to the API console page once token is generated.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Wizard.jsx [hunk 1]
```diff
@@ -308,13 +310,27 @@ class Wizard extends React.Component {
         this.setState({ value: event.target.value });
     };
 
+    /**
+     *
+     * Set state.redirect to true to redirect to the API console page
+     * @memberof Wizard
+     */
+    handleRedirectTest = () => {
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Wizard.jsx]: Shall we use the class method definition format, not this class member function style.
We have used this style before but don't recommend to use it anymore.
like

```javascript
class A {
    constructor() {
        this.myMethod = this.myMethod.bind(this);
    }

myMethod() {}
}
```

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx [hunk 2]
```diff
@@ -262,11 +262,20 @@ class Details extends React.Component {
...
+
+    /**
+     *
+     * Selects the active link for the side panel based on the URL
+     * @memberof Details
+     */
+    updateActiveLink() {
         const currentLink = this.props.location.pathname.match(/[^\/]+(?=\/$|$)/g);
-        if (currentLink && currentLink.length > 0) {
+        if (currentLink && currentLink.length > 0 && this.state.active !== currentLink[0]) {
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx]: `this.state.active` should have unloaded to a constant beforehand

```javascript
const {active} = this.state;
```

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Wizard.jsx (hunk 1 above)
Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Wizard.jsx]: Fixed

**Pull request merged**